WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -36,6 +36,9 @@ public PrivacySuggestionsFlyoutViewModel()
 
 	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, bool isFixedAmount, CancellationToken cancellationToken)
 	{
+		using CancellationTokenSource childCTS = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
```

Comment:
We often use this name imho:

```suggestion
		using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
```

I'm not sure that "child" is correct name here. If anything, it is more like "parent".

Reply:
I would prefer linkedCts

Reply:
[03c0f20](https://github.com/zkSNACKs/WalletWasabi/pull/7192/commits/03c0f202090b6fc74b496052730922c9c67785b3) commit fixes this.